cpp/src_prims/metrics/v_measure.cuh

```diff
@@ -41,7 +41,7 @@ double v_measure(const T* truthClusterArray,
                  T lowerLabelRange,
                  T upperLabelRange,
                  cudaStream_t stream,
-                 double beta = 1.0)
+                 double beta)
```

Comment:
Any specific reason we dropped the default value?

Reply:
If that default value is useful at the C++ layer, we may want to leave it in to maintain the best possible UX for C++ invocations. We'll want to keep in mind that a small but increasing number of applications are starting to look at invoking cuML directly at that layer.

Reply:
Got it. Shall I add it back then over here?

Reply:
Sure! Let's bring it back

Reply:
Done.